browser/brave_vpn/win/brave_vpn_helper/brave_vpn_helper_utils.cc

```diff
@@ -94,6 +94,11 @@ std::wstring GetBraveVpnHelperServiceName() {
   return name;
 }
 
+std::wstring GetBraveVpnHelperServiceDescription() {
+  // TODO(anyone): Consider adding a description.
```

Comment:
```suggestion
```

Reply:
Done in https://github.com/brave/brave-core/commit/9b86ae3f819b333a86283a06cc2faad00ba5abae